### PR TITLE
[Documentation]: Add google summer of code blog post

### PR DESCRIPTION
**Pull request by Freedisch**
## Description

This PR adds a blog post talking about my experience and work with google summer of Code

### Checklist

* [x] Test your changes as thoroughly as possible before you commit them. Preferably, automate your test by unit/integration tests.
* [x] Make sure that all your commits are signed-off and that you are added to the [Contributors](https://github.com/secureCodeBox/secureCodeBox/blob/main/CONTRIBUTORS.md) file.
* [x] Make sure that all CI finish successfully.
* [x] Optional (but appreciated): Make sure that all commits are [Verified](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits).

### CODE REVIEW

File: documentation/static/img/blog/2024-08-20-workflow.png.license [hunk 1]
```diff
@@ -0,0 +1,3 @@
+SPDX-FileCopyrightText: Look Up Look Down Photography
+
+SPDX-License-Identifier: LicenseRef-Unsplash-License
```
Comment (J12934) [inline documentation/static/img/blog/2024-08-20-workflow.png.license:3]: This should be like the apache 2 license we have in the other files

File: documentation/static/img/blog/2024-08-20-gsoc.png.license [hunk 2]
```diff
@@ -0,0 +1,3 @@
+SPDX-FileCopyrightText: Look Up Look Down Photography
+
+SPDX-License-Identifier: LicenseRef-Unsplash-License
```
Comment (J12934) [inline documentation/static/img/blog/2024-08-20-gsoc.png.license:3]: This should probably point out that this is not available under a open source license but the copyright belongs to google?

Comment (Freedisch) [reply documentation/static/img/blog/2024-08-20-gsoc.png.license:3]: For this, I made a screenshot of this image from their website, but kinda like being used everywhere

File: documentation/blog/2024-08-20-google-summer-of-code.md [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+
+**Output:**
+```bash
+Scans
+├── initial-nmap-scan
+│   ├── follow-up-vulnerability-scan
+│   │   └── detailed-sql-injection-scan
+└── another-initial-scan
+    └── another-follow-up-scan
+```
```
Comment (ddddddO) [inline documentation/blog/2024-08-20-google-summer-of-code.md:123]: Hi, @Freedisch  ! Thank you for using [gtree](https://github.com/ddddddO/gtree) for scbctl cascade! I'm very happy. (I'm the author of gtree)

I found this PullRequest and took a quick look, but maybe the output below is correct?

```bash
Scans
├── initial-nmap-scan
│   └── follow-up-vulnerability-scan
│       └── detailed-sql-injection-scan
└── another-initial-scan
    └── another-follow-up-scan
```

Also, I read this blog. Excellent!
If you want to improve the cascade subcommand, maybe [**WalkProgrammably** func](https://github.com/ddddddO/gtree?tab=readme-ov-file#walkprogrammably-func) is more useful than OutputProgrammably func.

The following link, which collects some tree libraries (and commands) other than gtree, might be helpful!
https://github.com/stars/ddddddO/lists/tree

Sorry for appearing out of nowhere! Bye!

Comment (Freedisch) [reply documentation/blog/2024-08-20-google-summer-of-code.md:123]: Thanks for the feedback : )
I will definitely take a look at the `WalkProgrammably func`

**Review by J12934 — APPROVED** [review]

**Pull request merged**